eternalcombat-api/src/main/java/com/eternalcode/combat/fight/FightManager.java

```diff
@@ -10,13 +10,13 @@
 import java.time.Instant;
 import java.util.Collection;
 import java.util.Collections;
+import java.util.HashMap;
 import java.util.Map;
 import java.util.UUID;
-import java.util.concurrent.ConcurrentHashMap;
 
 public class FightManager {
 
-    private final Map<UUID, FightTag> fights = new ConcurrentHashMap<>();
+    private final Map<UUID, FightTag> fights = new HashMap<>();
```

Comment:
What is causing this change?
Has this been tested against BossBar notifications? From what I remember, ConcurrentHashMap is there to avoid the ConcurrentModificationException error.

Reply:
Było to testowane? Mam wrażenie że po tej zmianie FightTask będzie właśnie rzucał concurrent error